account/account_test.go

```diff
@@ -526,10 +526,10 @@ func TestAddInvoke(t *testing.T) {
 		err = acnt.SignInvokeTransaction(context.Background(), &test.InvokeTx.InvokeTxnV1)
 		require.NoError(t, err)
 
-		resp, err := acnt.AddInvokeTransaction(context.Background(), test.InvokeTx)
+		resp, err := acnt.AddTransaction(context.Background(), test.InvokeTx)
 		if err != nil {
 			require.Equal(t, test.ExpectedErr.Error(), err.Error(), "AddInvokeTransaction returned an unexpected error")
-			require.Nil(t, resp)
+			require.Nil(t, resp.(*rpc.AddInvokeTransactionResponse))
```

Comment:
This and all other type assertions could be removed when the general Transaction Response type is ready